src/mpv_calendar/scripts/mpv-calendar-watcher

```diff
@@ -184,7 +184,7 @@ _watch_for_calendar_events () {
             # trick to get a non-zero return code if the media can't be played
             if echo "$title" | grep -q ':' || test -f "$title"
             then
-                if test "$no_default_image_fallback" -eq 0
+                if test "$default_image_fallback_calendar" == True
```

Comment:
Oops, this was a bug. This comes in as a string "True" or "False" not an integer.